### ISSUE TITLE
NewPipe is Importing subscriptions from my wrong YouTube account

### ISSUE THREAD

**Issue by Jannoth** [TeamNewPipe/NewPipe#2390]
Hi, I've worked out why my YouTube subscriptions are not being imported correctly. I watch YouTube on 2-3 Android devices.

I'm always logged into YouTube as same user and follow subscriptions etc but the Google account varies as that is linked to the device. I hope that makes sense.

My problem is that NewPipe imports subscriptions from the Google account associated with the device not a preferred YouTube account.

I haven't worked out what to do about it yet, but wondered if a login prompt during the import phase would be useful to other people too?? Or, should I do something else?

Thank you in advance and sorry for the headache haha!

J

**Comment by Stypox** [comment]
You have to log into your correct account from your browser, then you open the url to download subscriptions and you should get the correct file.

**Comment by abhishekshah67** [comment]
You must be downloading the subscriptions files from a browser (google chrome probably) where the id being used is your device id. You should open youtube in the browser and switch to desired account and then click the download link from newpipe for correct file. Make sure to remove the old files before downloading.

**Comment by Jannoth** [comment]
Thanks for your guidance.

To cut a long story short, I have successfully imported subscriptions from the correct account.

I also happen to use two browsers, so in both browsers I logged into the correct YouTube account then used the URL within the App to download the file.  I tried using just one browser (after changing the default) but it made no difference, so I simply logged into YouTube in both browsers.

It worked, so thank you again!

J 🙂
